### DISCUSSION TITLE
Selling 9x full kits V2B (UK)

### DISCUSSION THREAD

**Discussion by metaljay** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#580] [General]
Tested on my 2023 Ibiza square pump.
I have cables for both 4 and 6 wire pumps

drop me an email if interested.
Board + components + 4 & 6 wires = £20 (plus postage)

If you want me to solder and flash for you then £35 (plus postage) 

I hadn’t realised how much of a ball ache soldering the resistors were hence price.

contact: all sold
(iCloud hide my email address) 
few photos:

![IMG_1904](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/10380582/b13517c2-03fc-4e3b-ae5c-bae41cba0f22)
![IMG_1539](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/10380582/fb8a3c9c-1e13-4a4a-a4ed-2279f153d21d)
![IMG_1494](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/10380582/d9dafb40-d750-4d2d-ad8d-099b9f347114)

**Comment by Inazone** [comment]
Kits look good, just emailed you.

**Comment by metaljay** [comment]
Only 3 remain

**Comment by visualapproach** [comment]
Nice Crocs 👹

**Comment by Captain0Kangaroo** [comment]
Emailed you.

**Comment by metaljay** [comment]
I may order more if there is still a demand, I can ship world wide

**Comment by WesWesthaver** [comment]
Emailed you.

**Comment by Alexd911** [comment]
So your kits work with the Miami with the pump S100101?

Cheers
Alex

**Comment by piffmunday** [comment]
Hi, I am interested in one of these if you are going to do anymore. I have a Miama

**Comment by mpjanofsky** [comment]
I'd be interested in one of these if you make more - I'm in the USA.

**Comment by metaljay** [comment]
[For Sale] Complete kits or DIY (WorldWide) · visualapproach WiFi-remote-for-Bestway-Lay-Z-SPA · Discussion #615github.comRegardsJordan FernOn 23 Oct 2023, at 8:48 pm, Mike Janofsky ***@***.***> wrote:﻿
I'd be interested in one of these if you make more - I'm in the USA.
